documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -26,6 +26,8 @@
 
 * A new property, [detect.stateless.policy.check.fail.on.severities](properties/basic-properties.html#ariaid-title34) has been added, which will trigger [detect_product_short] to fail the scan and notify the user if a policy violation matches the configured value. This property overrides the default "Blocker" and "Critical" severity settings that cause [detect_product_short] scans to exit. This property applies to both [Rapid](runningdetect/rapidscan.md) and [Stateless](runningdetect/statelessscan.md) scans. Intelligent persistent scans, (when scan mode is not set to RAPID, STATELESS, or [--detect.blackduck.scan.mode](properties/all-properties.html#ariaid-title5) is explicitly set to INTELLIGENT and scan data is persisted), should continue using the [detect.policy.check.fail.on.severities](properties/basic-properties.html#ariaid-title34), property.
 
+* [detect.policy.check.fail.on.names](properties/configuration/project.md#fail-on-policy-names-with-violations) will now print the names of fatal policy violations at the Detect Status stage. 
```

Comment:
Tweak suggestion:
* [detect_product_short] will now print the names of fatal policy violations at the [detect_product_short] status stage if [detect.policy.check.fail.on.names](properties/configuration/project.md#fail-on-policy-names-with-violations) is configured.